checkbox-support/checkbox_support/parsers/udevadm.py

```diff
@@ -256,7 +256,10 @@ def category(self):
                      self._environment["INTERFACE"].startswith('wlan')
                      )):
                     return "WIRELESS"
-                if (self._environment["INTERFACE"].startswith('can')):
+                # generic CAN device will be named as 'can$n'
+                # LLCE CAN device will be named as 'llcecan$n'
+                if (re.search(r"^([a-z]+can|can[a-z]*)[0-9]+$",
```

Comment:
Is there any reason why you are not very specific with the `llcecan` part?

Also, do we have any examples where the interface name for a socketcan device is something else than `can$n`? If not, how about using `^((llce)*can)[0-9]+$` instead?

Reply:
Ok, I have sent a new commit for this issue.